libs/game/docs/reference/sprites/sprite/lifespan.md

```diff
@@ -0,0 +1,66 @@
+# lifespan (property)
+
+Get or set the lifespan of sprite in update units.
+
+## Get
+
+Get the lifespan of the sprite.
+
+```block
+let mySprite: Sprite = null
+
+let lifetime = mySprite.lifespan
+```
+
+```typescript-ignore
+let lifetime = mySprite.lifespan
+```
+
+### Returns
+
+* a [number](/types/number) that is the current lifespan of the sprite.
+
+## Set
+
+```block
+let mySprite: Sprite = null
+
+mySprite.lifespan = 0
+```
+
+```typescript-ignore
+mySprite.lifespan = 0
+```
+
+### Parameter
+
+* **value**: the new lifespan of the sprite.
+
+## Sprite lifespan
+
+Sprites that aren't set to auto destroy will remain in the game until they are destroyed by calling their **destroy** function or until their lifespan expires. You can set the lifespan of sprite to make it leave the game after some amount of time.
```

Comment:
Can we change the wording to _until their lifespan expires (if specified)_?

Reply:
Yea, it's good to clarify that. Thanks